.github/workflows/django.yml

```diff
@@ -38,7 +38,7 @@ jobs:
       continue-on-error: ${{ matrix.django-version == '3.1.0' || matrix.django-version == 'main' }}
       run: |
         python -m pip install --upgrade pip
-        pip install psycopg2
+        pip install 'psycopg2<2.9'
```

Comment:
Could we add a comment here linking to the bug and explaining that we can drop this requirement once we drop support for Django 2.2?

I guess the nicest solution is to only use `pyscopg2 < 2.9` for the Django 2.2 tests, but let's not do that in this PR.

Reply:
3.0 is also affected. I updated the PR to restrict the pin to Django < 3.1

Reply:
Hmm. If this also affects 3.0, we should probably tell the Django and psycopg2 developers in the bug you linked to -- psycopg/psycopg2#1293 -- since 3.0 is a lot further from end of life.

Reply:
Django 3.0 wasn't an LTS release, so it hit EOL in April

We need to update the set of tested Django versions, although that can probably wait until we add python 3.10 to the mix as well.